Android/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required (VERSION 2.8.7)
+project(MCServer)
+include(ExternalProject)
+
+externalproject_add(lua_native
+	SOURCE_DIR ../lib/lua
+	CMAKE_GENERATOR "Visual Studio 14 2015"
```

Comment:
It's definitely not right, but conversely, I have no idea how to get the autodetection working as it does when you run `cmake .`